src/constants.ts

```diff
@@ -27,7 +27,7 @@ export enum RuntimeType {
   UNSUPPORTED,
 }
 
-export const DefaultDatadogProps = {
+export const DefaultDatadogLambdaProps = {
```

Comment:
I kinda prefer `DatadogLambdaDefaultProps` but only because it falls in line with the naming of the other props types

Reply:
I couldn't find another default prop type. Could you give an example?

Reply:
Sorry, I think I wasn't clear. My (nitpick) preference is that this prop be renamed `DatadogLambdaDefaultProps` rather than `DefaultDatadogLambdaProps`. The other prop types I was referring to were:

`DatadogLambdaStrictProps`  
`DatadogLambdaProps`

Notice that these both start with `Datadog`, where as the default one doesn't.

Reply:
Make sense! Will rename it.